index.html

```diff
@@ -0,0 +1,288 @@
+                        <li><div id="work4tag3"><div id="javascript4">javascript</div></div></li>
+                    </ul>
+                </div>
+                <div class="work4Actions">
+                    <button id="work4see-project">
+                       <b>See Project</b>
+                    </button>
+                </div>
+               </div>
+               
+            </section>
+        </section>
+        <section class="About-myself">
+            <section class="top">
+                <div id="primary-heading"><h2>About Myself</h2></div>
+                <div id="primary-paragraph">
+                    <p>Hello I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    </div>
+                    <div class="lets-connect">LET’S CONNECT</div>
+                    <div class="connect-icons">
+                        <ul>
+                            <li><a href="https://github.com/Kevo-hacker"><img src="github.png" alt=""></a></li>
+                            <li><a href="https://www.linkedin.com/in/kelvin-ben-323043173/"><img src="Linkedin12.png" alt=""></a></li>
+                            <li><a href="peace.com"><img src="peace.png" alt=""></a></li>
+                            <li><a href="https://twitter.com/kevoyout144"><img src="twitter.png" alt=""></a></li>
+                            <li><a href="Gmail.com"><img src="mail.png" alt=""></a></li>
+                        </ul>
+                    </div>
+                     <button id="my-resume"><span>Get my resume</span></button>
+            </section>
+            <div class="skill-list">
+                    <div id="frame12">
+                        <div id="language">
+                            <div id="drop-down">
+                                <img src="Chevron-Down.png" alt=""></div>
+                                <div id="lang">
+                                    <h5>Languages</h5>
+                                </div>
+                            
+                        </div>
+                    </div>
+                    <div id="lang-list">
+                        <div id="frame9">
+                            <div id="img"><img src="javascript.png" alt=""></div>
+                            <div id="js">javascript</div>
+                            
+                        </div>
+                    </div>
+                    <div id="frame10">
+                        <div id="img1"><img src="html.png" alt=""></div>
+                            <div id="html">HTML</div>
+                    </div>
+                    <div id="frame11">
+                        <div id="img2"><img src="css.png" alt=""></div>
+                            <div id="css">CSS</div>
```

Comment:
[OPTIONAL]

- [ ] I noticed that you placed your images in the root folder, that's one way to solve the issues the previous reviewers mentioned but the optimal way to solve it is to place the images in one folder again, and then use what is called a relative path, a relative path is the same as the usual path just you need to add `./` at the beginning of it for example `./src/image1`, to give you an idea what this relative path is, a relative Path is the hierarchical path that locates a file or folder on a file system starting from the current directory, this is the best way to locate your images correctly and prevent any issues from happening! 😁